randovania/server/app.py

```diff
@@ -8,20 +8,56 @@
 
 import randovania
 import randovania.server.multiplayer.world_api
+from randovania.network_common import error
 from randovania.server import user_session, database, client_check, multiplayer
 from randovania.server.multiplayer import world_api
 from randovania.server.server_app import ServerApp
 
 
+class CustomFormatter(logging.Formatter):
```

Comment:
surely we can come up with a more descriptive name than this

Reply:
I accept suggestions :p

Reply:
`ServerLoggingFormatter`?